MakingMusicTool/MakingMusicTool.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @Date    : 2021-05-07 16:05:35
+# @Author  : Tom Brandherm (tom.brandherm@msasafety.com)
+# @Link    : link
+# @Version : 1.0.0
+"""
+Making music with python is possible by using this tool. As a import pattern 
+please use a created csv file with one or more rows like:
+
+[one of the given instruments],[volume from 1 to 3],[music note 1],...[music note n]
+
+A 4/4 time is used here, so the sum of the notes in one row have to be equals one.
+The notation for the notes into the csv is: 
+
+1/16    =>  16
+1/8     =>  8
+1/4     =>  4
+1/2     =>  2
+1/1     =>  1
+"""
+# =========================================================================== #
+#  Copyright 2021 Team Awesome
+# =========================================================================== #
+#  All Rights Reserved.
+#  The information contained herein is confidential property of Team Awesome.
+#  The use, copying, transfer or disclosure of such information is prohibited
+#  except by express written agreement with Team Awesome.
+# =========================================================================== #
+
+# =========================================================================== #
+#  SECTION: Imports                                                           
+# =========================================================================== #
+import os
+import csv
+
+from pydub import AudioSegment
+from pydub.playback import play
+# =========================================================================== #
+#  SECTION: Global definitions
+# =========================================================================== #
+# name of the input csv file
+FILENAME = "sound_pattern.csv"
+# =========================================================================== #
+#  SECTION: Class definitions
+# =========================================================================== #
+
+
+class MusicMaker(object):
+    """
+    MusicMaker is a class to that can create soundtracks out of a given pattern.
+    The pattern needs information about a instrument, the volume and a 4/4-time
+    rythm.
+    """
+    # ----------------------------------------------------------------------- #
+    #  SUBSECTION: Constructor
+    # ----------------------------------------------------------------------- #
+
+    def __init__(self, fileName: str, bpm:int=60):
+        ## public
+        # input from the csv file
+        self.sound_pattern = self.analyse_data(fileName)
```

Comment:
because self.analyse_data(fileName) returns something I would use get in the function name. 
Furthermore the comment above is unnecessary if the function name describes what the function does.
```suggestion
        self.sound_pattern = self.get_sound_pattern_from_csv_file(fileName)
```